orb/commands/e2e_start_command_runner.yml

```diff
@@ -3,14 +3,12 @@ parameters:
   command_runner_image:
     description: "The image to execute commands from against the kind cluster."
     type: string
-    default: "quay.io/reactiveops/ci-images:v10.0-alpine"
+    default: "quay.io/reactiveops/ci-images:v10-alpine"
 steps:
   - run:
       name: Setup Command Runner
       command: |
-        kubeconfig="$(kind get kubeconfig-path --name="e2e")"
-        echo $kubeconfig
-        cat $kubeconfig
+        cat ./kind-kubeconfig
```

Comment:
is `kind-kubeconfig` a new thing with kind?

Reply:
Kind now merges the kubeconfig into your existing one.  I didn't want to accidentally merge with anything, so it creates a new kubeconfig in the working directory.  You'll notice that I pass a flag when creating the cluster in order to do this.